sys-utils/ipcmk.c

```diff
@@ -44,6 +48,23 @@ static int create_shm(size_t size, int permission)
 	return shmget(key, size, permission | IPC_CREAT);
 }
 
+static int create_posix_shm(const char *name, size_t size, int permission)
+{
```

Comment:
Probably add `#ifdef HAVE_SHM_OPEN` (you have added to ./configure) for case the function is not supported, and return -ENOSYS in this case.

Reply:
Do not use the compiler command line in meson (e.g., `-DHAVE_SEM_OPEN`); instead, use config.h to store the setting. 
```
conf.set('HAVE_SHM_OPEN', have_sem_open ? 1 : false)
```